users/user-creation.md

```diff
@@ -0,0 +1,42 @@
+## User Creation
+
+Aside from [automatic account creation](/jetpack/automatic-account-creation-connection.md) there is an endpoint available for creating a WordPress.com user. This endpoint assumes the user creation is part of a product provisioning flow.
+
+
+### Endpoint Information (/user)
+
+- __Method__: POST
+- __URL__:    `https://public-api.wordpress.com/rest/v1.3/jpphp/user`
+
+### Request Parameters (/user)
+
+- __email__: The email address for the user that will be created.
+- __product_id__: The product id that is being provisioned for the user being created.
+
+
+### Response Parameters (/user)
+
+Below, the responses are grouped by whether the call to create a user was succesful or not.
+
+#### Successful response
+
+- __success__:       (bool) Was the operation successful?.
```

Comment:
It doesn't look like Github respects the spaces between the `:` and `(bool)` so, we should probably get rid of them here. Also, looks like it doesn't quite align with the bearer_token list item below.